addons/mail/static/src/core/common/mail_core_common_service.js

```diff
@@ -28,8 +28,7 @@ export class MailCoreCommon {
                 if (!message) {
                     continue;
                 }
-                this.env.bus.trigger("mail.message/delete", { message, notifId });
```

Comment:
`mail.message/delete` is another flow (actual hard delete) everything related to that should be left intact